packages/gasket-cli/lib/index.d.ts

```diff
@@ -187,12 +288,14 @@ export interface CreateContext {
 
 declare module '@gasket/engine' {
   export interface HookExecTypes {
-    initOclif(args: { oclifConfig: Config }): MaybeAsync<void>;
+    getCommandOptions(config: Config): MaybeAsync<Array<GasketOptionDefinition>>;
+
+    getCommands(config: Config): MaybeAsync<Array<GasketCommandDefinition> | GasketCommandDefinition>;
```

Comment:
With the pending new `actions` feature, were a prefix verb will be included, I'm wondering if these lifecycle names should just be `commands` and `commandOptions`? Leaving off a verb is more inline with the rest of existing lifecycle names.

Reply:
I'm open to more discussion, but it seems like a good alignment opportunity now.

Reply:
I'm open to this change. It's an involved change that touches a lot of files but I like it. Makes sense to me.